CHANGELOG.md

```diff
@@ -2,6 +2,78 @@
 
 All notable changes to this project are documented in this file.
 
+## 1.0.0-rc.1
+
+**Release date:** 2023-03-30
+
+This release candidate promotes the `GitRepository` API from `v1beta2` to `v1`.
+The controller now supports horizontal scaling using
+sharding based on a label selector.
+
+In addition, support for Azure Workload Identity was added to
+`OCIRepositories`, `Buckets` and `HelmRepositories` when using `provider: azure`.
+
+### Highlights
+
+#### API changes
+
+The `GitRepository` kind was promoted from v1beta2 to v1 (GA) and deprecated fields were removed.
+
+The common types `Artifact`, `Conditions` and the `Source` interface were promoted to v1.
+
+The `gitrepositories.source.toolkit.fluxcd.io` CRD contains the following versions:
+- v1 (storage version)
+- v1beta2 (deprecated)
+- v1beta1 (deprecated)
+
+#### Upgrade procedure
+
+The `GitRepository` v1 API is backwards compatible with v1beta2 with the following exceptions:
+- the deprecated field `.spec.gitImplementation` was removed
+- the unused field `.spec.accessFrom` was removed
+- the deprecated field `.status.contentConfigChecksum` was removed
+- the deprecated field `.status.artifact.checksum` was removed
+- the `.status.url` was removed in favor of the absolute `.status.artifact.url`
```

Comment:
```suggestion
The `GitRepository` v1 API is backwards compatible with v1beta2, except for the following:

- the deprecated field `.spec.gitImplementation` was removed
- the unused field `.spec.accessFrom` was removed
- the deprecated field `.status.contentConfigChecksum` was removed
- the deprecated field `.status.artifact.checksum` was removed
- the `.status.url` was removed in favor of the absolute `.status.artifact.url`
```

maybe to prevent "with ... with"?